brave/browser-laptop
# Revert "Hide Enpass until it's supported in muon"

This reverts commit 29a8223c19a8127f195bf0c3bd46ebea1dbe5270.

According to issue #7778, particularly the comment by @mcg, support for Enpass is already working. According to @bsclifton, in the same thread, we should just create the pull request for it. So I thought I'd just go ahead and do it :)

I rebuilt and enabled on my machine (OS X) and it worked fine.

Submitter Checklist:


Test Plan:

- Enable enpass password manager
- Autofill some passwords
- Check that it doesn't crash

Reviewer Checklist:

Tests
